.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v2
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+
+      - name: Run the Maven verify phase
+        run: mvn --batch-mode --update-snapshots verify
```

Comment:
Copy this from the GitHub Action docs. I think that those guys known the idiomatic way, but here explanations of the switches:
- [`--batch-mode`](https://books.sonatype.com/mvnref-book/reference/running-sect-options.html#running-sect-batch-option) -- do not request any interactive input if it can arise
- [`--update-snapshots`](https://books.sonatype.com/mvnref-book/reference/running-sect-options.html#running-sect-deps-option) -- needed only if we have dependencies bu we don't. Will remove
- [`verify`](https://maven.apache.org/guides/introduction/introduction-to-the-lifecycle.html#usual-command-line-calls) -- runs almost all phases of the project except actual installing into local repository and deploying to the remote repository. It seems ideal for tests